sources/packages/web/src/views/aest/institution/InstitutionDetails.vue

```diff
@@ -5,9 +5,8 @@
     </div>
     <span class="heading-x-large mb-2">
       {{ institutionBasicDetail.operatingName }}
-      <designation-and-restriction-status-badge
-        class="mb-4 ml-4"
-        status="designated"
+      <status-chip-designation-agreement-location
+        :status="institutionBasicDetail.designationAgreementLocationStatus"
```

Comment:
IMO we should avoid using the location in this context.
It would be like below.
`institutionBasicDetail.designationAgreementStatus`